src/__stories__/menu-story.tsx

```diff
@@ -49,6 +58,38 @@ export const Default: StoryComponent = () => {
                     </div>
                 )}
             />
+            <SectionTitle>Menu with close option</SectionTitle>
+            <Menu
+                width={200}
+                renderTarget={({ref, onPress, isMenuOpen}) => (
+                    <Touchable elementRef={ref} onPress={onPress}>
+                        <Inline space={16}>
+                            <IconKebabMenuLight />
+                            <Text3 regular>{isMenuOpen ? 'Close' : 'Open'}</Text3>
+                        </Inline>
+                    </Touchable>
+                )}
+                renderMenu={({ref, className, close}) => (
+                    <div ref={ref} className={className}>
+                        {fruitOptions.map((fruit) => (
+                            <Box paddingX={16} key={fruit.text} paddingY={8}>
+                                <Touchable
+                                    onPress={() => {
+                                        close();
+                                        confirm({
+                                            title: 'Title',
+                                            message: 'This is the dialog message',
+                                            icon: <IcnInfo />,
+                                        });
```

Comment:
why?

Reply:
@pladaria ask me to add a story with the use of this new prop

Reply:
yes, I'm talking about show a modal that will freeze the UI each time I'm playing with Menu component

Reply:
you can already check the last version deployed https://mistica-web-4s5z16vd2-atabel.vercel.app/?path=/story/components-controls-menu--default